src/test/java/ru/levelup/elena/roshchina/qa/homework_07/AbstractGoRestAPITest.java

```diff
@@ -0,0 +1,116 @@
+package ru.levelup.elena.roshchina.qa.homework_07;
+
+import org.testng.annotations.BeforeMethod;
+import ru.levelup.elena.roshchina.qa.homework_07.comments.CommentsPojo;
+import ru.levelup.elena.roshchina.qa.homework_07.comments.SingleComment;
+import ru.levelup.elena.roshchina.qa.homework_07.posts.PostsPojo;
+import ru.levelup.elena.roshchina.qa.homework_07.posts.SinglePost;
+import ru.levelup.elena.roshchina.qa.homework_07.users.SingleUser;
+import ru.levelup.elena.roshchina.qa.homework_07.users.UsersPojo;
+
+import java.io.*;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+
+public abstract class AbstractGoRestAPITest {
+    protected final String APIToken = "e2735da4aa6087a53f5374a398698fbf5aee5eb48da6944b899cc433ce9e1d2a";
+    protected ArrayList<Map<String, String>> userData;
+    protected ArrayList<Map<String, String>> postData;
+    protected ArrayList<Map<String, String>> commentData;
+    private final String userDataFileName = "user_data.csv";
+    private final String postDataFileName = "post_data.csv";
+    private final String commentsDataFileName = "comment_data.csv";
+    private URL dataFilePathName = this.getClass().getResource("/ru.levelup.elena.roshchina.qa/homework_07/");
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.